frontend/gettext.lua

```diff
@@ -51,7 +51,7 @@ function GetText_mt.__index.changeLang(new_lang)
     local po = io.open(file, "r")
 
     if not po then
-        logger.warn("cannot open translation file:", file)
+        logger.dbg("cannot open translation file:", file)
```

Comment:
This string is displayed early at start, just after device init. On some crashes this is the last string you'll get on the logs, not just on Android.  I'm not sure if this is a matter of personal preference or what. Anyways, it can be left as a warning...

Reply:
My comment isn't about the change itself, but about using `git blame` in the future. ;-)